net/ipvsadm/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2016-2017 Mauro Mozzarelli
+#
+# This is free software, licensed under the GNU General Public License
+# See /LICENSE for more information.
+#
+# AUTHOR: Mauro Mozzarelli <mauro@ezplanet.org>
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ipvsadm
+PKG_VERSION:=1.29
+PKG_MAINTAINER:=Mauro Mozzarelli <mauro@ezplanet.org>
+PKG_LICENSE:=GPL-1.0
```

Comment:
I am not sure in this case.  It looks suspicious to me.  `rpm -qi ipvsadm` on centos7 says it should be GPL-2.0, can you confirm that?

```
Name        : ipvsadm
Version     : 1.27
Release     : 7.el7
Architecture: x86_64
Install Date: Thu 20 Sep 2018 12:51:50 PM UTC
Group       : Applications/System
Size        : 76749
License     : GPLv2+
Signature   : RSA/SHA256, Sat 14 Mar 2015 07:56:16 AM UTC, Key ID 24c6a8a7f4a80eb5
Source RPM  : ipvsadm-1.27-7.el7.src.rpm
```

Reply:
You are right see http://svn.linuxvirtualserver.org/repos/ipvsadm/trunk/README
I will change this to GPLV2+